scripts/readme.md

```diff
@@ -110,20 +123,38 @@ Run this using:
 
 ```
 register=jobcentre-district
-new_description='bla bla bla bla bla bla'
+desc='bla bla bla bla bla bla'
 
-python update_descriptions.py $register $new_description > ${register}_update.rsf
+python update_descriptions.py $register $desc
 ```
 
-Then load the RSF using:
+Then check that the RSF looks correct.
+
+To actually load the RSF, pipe the output to `rsf-load.sh`:
 
 ```
-./rsf-load.sh "https://${register}.beta.openregister.org" openregister `register-pass beta/app/mint/$register` < ${register}_update.rsf
+python update_descriptions.py $register $desc | ./rsf-load.sh "https://${register}.beta.openregister.org" openregister `registers-pass beta/app/mint/$register` < ${register}_update.rsf
 ```
 
 This example assumes that the register is in beta.
 
-This will update the API explorer, but at the time of writing, registers frontend won't automatically pick up the description, because system entries are not included in incremental updates.
+This will update the API explorer, but at the time of writing, registers frontend won't automatically pick up the description, because system entries are not included in incremental updates. To do this, follow the [instructions for redownloading a register in registers frontend](https://github.com/openregister/registers-frontend#populating-the-database-with-register-data-on-paas).
```

Comment:
If you want, this script could also perform the re-index after updating e.g.
```
cf run-task registers "bundle exec rake registers_frontend:populate_db:force_full_register_download[$register]" -m 1G --name force_full_reindex
```
we would need to make sure you were logged in and in the correct space though.

Reply:
I think I'll leave this bit out for now, because it depends on our local environments being set up in a particular way, and I'd rather run this part one-by-one anyway.

This time round I messed up the frontend by running this, so I think it makes sense to test populating the registers locally before doing it in production.